2021-07-01 22:02:01,225 - INFO - ServerEntity.LoadCollector - [ServerEntity.py:25]: __init__ create entity LoadCollector with id 60ddcad99027cfb41edd04f2 mem_id=2184702453264
2021-07-01 22:02:01,227 - INFO - server.EntityManager - [EntityManager.py:139]: addentity  entity id  60ddcad99027cfb41edd04f2 
2021-07-01 22:02:01,238 - INFO - ServiceNode - [EtcdSupport.py:549]: we have create ServiceRegister
2021-07-01 22:02:01,240 - INFO - ServiceNode - [EtcdSupport.py:552]: we have create ServiceFinder
2021-07-01 22:02:01,250 - INFO - TcpServer - [TcpServer.py:200]: Server on ('127.0.0.1', 9101)
2021-07-01 22:02:04,250 - INFO - ServiceFinder - [EtcdSupport.py:382]: _init_info, now_url: http://127.0.0.1:2379/v2/keys?recursive=true
2021-07-01 22:02:04,254 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> dispatcher_service: ('127.0.0.1', 9102)
2021-07-01 22:02:04,255 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> dispatcher_service: ('127.0.0.1', 9100)
2021-07-01 22:02:04,257 - INFO - ServiceFinder - [EtcdSupport.py:353]: add service node info--> dispatcher_service: ('127.0.0.1', 9101)
2021-07-01 22:02:04,258 - INFO - ServiceFinder - [EtcdSupport.py:302]: init service info from etcd success, will going to watch, at etcd index -> 388
2021-07-01 22:02:20,862 - INFO - TcpServer - [TcpServer.py:186]: ('127.0.0.1', 5475) is connected !!!!
2021-07-01 22:02:39,861 - INFO - ServiceFinder - [EtcdSupport.py:464]: recv watch expire, info--> dispatcher_2: ('127.0.0.1', 9102) - [417]
2021-07-01 22:02:40,362 - INFO - ServiceFinder - [EtcdSupport.py:464]: recv watch expire, info--> dispatcher_0: ('127.0.0.1', 9100) - [418]
2021-07-01 22:06:17,291 - WARNING - TcpConn - ["C:\Users\b\Documents\github\realtime-server\pycharm2020.1.3\script\TcpConn.py:168"]: peer addr: ('127.0.0.1', 5475), close_reason='connection is closed by error: [WinError 64] The specified network name is no longer available'
